libs/ui/src/reports/certification_signatures.tsx

```diff
@@ -3,11 +3,13 @@ import { Font } from '../typography';
 
 const CertificationSignaturesContainer = styled.div`
   margin-top: 1em;
+  background-color: #e8e8e8;
+  border: 1px solid rgb(194, 200, 203);
+  padding: 0.5em;
```

Comment:
Starting a thread here to discuss @adghayes feedback:
> I'm skeptical of the shaded card for the certification signatures. It seems like it might make it harder to read the signatures. Are there other container options here that might work?

Glad to hear your feedback on this - I meant to ask for feedback on it specifically because I had a similar wondering. Your question spurred me to actually test it. I was pleasantly surprised - it didn't seem to impact the readability of the signatures at all. In the photo below, you'll see both pen (signature 1) and pencil (signature 2), with pen below on white for comparison.

![IMG_9085](https://github.com/user-attachments/assets/af0263d5-3ac2-44a5-bd26-2251ee16138d)

This is enough to assuage the concern for me, but I'm still open to exploring other options if you'd like.

I originally tried a box with a border but no background, but it didn't look great. Which reminds me - is that how this will print from the VxScan thermal printer? It might be worth a test to check how that looks.

Reply:
I agree that looks fine!

But the thermal printer question is the actually good question here... I've attached an image of a thermal report from my camera roll. If the greys are the same as the "Ballot Counts" header right below it, it'll print as in the picture - white background black border. So - no background. If you think that'll look bad, we could do a different approach. I defer to you on the aesthetics as long as what we do doesn't take up vertical space beyond what's taken up on `main`.

![IMG_1022 2](https://github.com/user-attachments/assets/dddf6825-c43b-4228-a7dd-4da59f326a68)

Reply:
It actually looks fine with the white background, so I'm going to leave it as is